### ISSUE TITLE
rocon 1.2.4 causes compile error

### ISSUE THREAD

**Issue by mtgto** [uhyo/rocon#28]
This code causes compile error using rocon v1.2.4.
With rocon v1.2.3, it does not happen compile error.

```ts
// Copy from https://rocon.uhyohyo.net/tutorial/navigation
import Rocon, { useNavigate } from "rocon/react";
import React from "react";

const toplevelRoutes = Rocon.Path()
  .any("id", {
    action: ({ id }) => <p>Hello, {id}</p>
  });

const navigate = useNavigate();
navigate(toplevelRoutes.anyRoute, { id: "uhyo" });
```

```console
❯ yarn run tsc --noEmit
yarn run v1.22.17
$ /Users/user/work/js/example-rocon/node_modules/.bin/tsc --noEmit
index.tsx:10:35 - error TS2554: Expected 1 arguments, but got 2.

10 navigate(toplevelRoutes.anyRoute, { id: "uhyo" });
                                     ~~~~~~~~~~~~~~


Found 1 error.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

https://gist.github.com/mtgto/36d02086138f37ed368250dfe47802cf

**Comment by uhyo** [comment]
Thank you very much for reporting this! 🥺

**Comment by uhyo** [comment]
Fixed in v1.2.5.

**Comment by mtgto** [comment]
Thank you for your quick response 😍
